assets/js/components/legacy-notifications/index.js

```diff
@@ -39,7 +39,17 @@ const notification = getQueryParameter( 'notification' );
 const addCoreSiteNotifications = createAddToFilter( <DashboardCoreSiteAlerts /> );
 const addSetupNotifications = createAddToFilter( <DashboardSetupAlerts /> );
 const addModulesNotifications = createAddToFilter( <DashboardModulesAlerts /> );
-const addUserInputSettings = createAddToFilter( <UserInputSettings /> );
+const addUserInputSettings = createAddToFilter(
+	<UserInputSettings
+		isDismissable={ true }
+		onDismiss={ () => {
+			trackEvent( 'user_input', 'prompt_notification_dismiss' );
+		} }
+		onCTAClick={ () => {
+			trackEvent( 'user_input', 'prompt_notification_start' );
+		} }
+	/>
```

Comment:
Let's introduce a new `UserInputPromptNotification` for this logic here. We should then also include the `prompt_notification_view` event in there (see my other comment).

Reply:
👍